apps/console/src/index.html

```diff
@@ -23,15 +23,14 @@
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
         <meta name="referrer" content="no-referrer" />
 
+        <link href="<%= htmlWebpackPlugin.options.publicPath %>/libs/themes/default/theme.<%= htmlWebpackPlugin.options.themeHash %>.min.css" rel="stylesheet" type="text/css"/>
```

Comment:
Lets revert the changes on the `Helmet` as well. Else the theme will be loaded twice.